elements/lisk-transactions/src/utils/create_base_transaction.ts

```diff
@@ -37,3 +40,17 @@ export const createBaseTransaction = ({
 		senderPublicKey,
 	};
 };
+
+export const serializeSignatures = (signatures: ReadonlyArray<string>) => {
+	const signaturesBuffer = signatures.map(signature => {
```

Comment:
- This can be achieved with only one loop (you can check protocol-specs for multisig for the same functionality)
- it would be ideal to use variables for the `00` and `01` buffers so they make more sense

Reply:
I had a look at your reference @pablitovicente, I addressed the variables part, but the serialization logic looks clear and good to me. Let me know if it is still a concern.